public/static/app/waterfall/waterfall.jsx

```diff
@@ -650,6 +658,7 @@ function ActiveVersionHeader({shortenCommitMessage, version, onLinkClick, userTz
             <div className="row">
               <strong>{author}</strong> - <JiraLink jiraHost={jiraHost}>{message}</JiraLink>
               {button}
+              <div>{tags}</div>
```

Comment:
You don't need to declare tags as a variable, you can just put code inside the braces